docs/source/llms/llm-evaluate/index.rst

```diff
@@ -222,65 +209,209 @@ for LLM evaluation in MLflow. You can specify a custom list of metrics in the `e
 
-To use an endpoint hosted by a local MLflow AI Gateway, you can use the following code.
+1. Start the MLflow AI Gateway server with your LLM setting by following `these steps <../deployments/index.html#quickstart>`_.
+2. Set MLflow deployment client to target the server address by using :py:func:`~mlflow.deployments.set_deployments_target()`.
+3. Set ``endpoints:/<endpoint-name>`` to the ``model`` parameter in the metrics definition.
 
 .. code-block:: python
 
     from mlflow.deployments import set_deployments_target
 
+    # When the MLflow AI Gateway server is running at http://localhost:5000
     set_deployments_target("http://localhost:5000")
     my_answer_similarity = mlflow.metrics.genai.answer_similarity(
         model="endpoints:/my-endpoint"
     )
 
-To use an endpoint hosted on Databricks, you can use the following code.
+4. Databricks Model Serving
+###########################
+
+If you have a model hosted on Databricks, you can use it as a judge model by setting ``endpoints:/<endpoint-name>`` to the ``model`` parameter in the metrics definition. The following code uses a Llama 3.1 405B model available via `Foundation Model API <https://docs.databricks.com/en/machine-learning/model-serving/index.html>`_.
 
 .. code-block:: python
 
     from mlflow.deployments import set_deployments_target
 
     set_deployments_target("databricks")
-    llama2_answer_similarity = mlflow.metrics.genai.answer_similarity(
-        model="endpoints:/databricks-llama-2-70b-chat"
+    llama3_answer_similarity = mlflow.metrics.genai.answer_similarity(
+        model="endpoints:/databricks-llama-3-1-405b-instruct"
+    )
+
+Overriding Default Judge Parameters
+***********************************
+
+By default, MLflow queries the judge LLM model with the following parameters:
+
+.. code-block:: yaml
+
+    temperature: 0.0
+    max_tokens: 200
+    top_p: 1.0
+
+However, this might not be suitable for all LLM providers. For example, accessing Anthropic's Claude models on Amazon Bedrock requires ``anthropic_version`` parameter to be specified in the request payload. You can override these default parameters by passing the ``parameters`` argument to the metrics definition.
+
+.. code-block:: python
+
+    my_answer_similarity = mlflow.metrics.genai.answer_similarity(
+        model="bedrock:/anthropic.claude-3-5-sonnet-20241022-v2:0",
+        parameters={
+            "temperature": 0,
+            "max_tokens": 256,
+            "anthropic_version": "bedrock-2023-05-31",
+        },
     )
 
-For more information about how various models perform as judges, please refer to `this blog <https://www.databricks.com/blog/LLM-auto-eval-best-practices-RAG>`_.
+Note that the parameters dictionary you pass in the ``parameters`` argument will replace the default parameters, instead of being merged with them. For example, ``top_p`` will **not** be sent to the model in the above code example.
```

Comment:
```suggestion
Note that the parameters dictionary you pass in the ``parameters`` argument will **replace the default parameters** instead of being merged with them. For example, ``top_p`` will **not** be sent to the model in the above code example.
```